.github/workflows/pr.yml

```diff
@@ -46,7 +46,7 @@ jobs:
 
     - name: Generate test cases
       working-directory: ./test/Esprima.Tests.Test262
-      run: dotnet tool restore && dotnet test262 generate
+      run: dotnet tool restore && dotnet test262 generate && dotnet run -c Release -- --update-allow-list
```

Comment:
If that updates the allow-list, then this is just saying "anything that doesn't work is ignored", right? So if I create a PR that only breaks a unit test, would it just be ignored and hide it from us?

Reply:
> Also running allow list check as part of build, it will return non-zero exit code if allow-list is out of date